xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitlesLibass.cpp

```diff
@@ -196,7 +412,129 @@ ASS_Event* CDVDSubtitlesLibass::GetEvents()
 int CDVDSubtitlesLibass::GetNrOfEvents() const
 {
   CSingleLock lock(m_section);
-  if(!m_track)
+  if (!m_track)
     return 0;
   return m_track->n_events;
 }
+
+int CDVDSubtitlesLibass::AddEvent(const char* text, double startTime, double stopTime)
+{
+  if (!*text)
```

Comment:
I think this is unsafe as you are blindingly dereferencing the pointer. To check if the "string" is empty I think you need to check if the pointer is null and then check if the first char is the string delimiter ('\0')

Reply:
can go well as `if (text == NULL || text[0] == '\0')` ?

Reply:
Yeah I think that's the correct way

Reply:
done